keps/sig-multicluster/4322-cluster-inventory/README.md

```diff
@@ -379,14 +382,38 @@ command or function to work in the same way in another tool. This can
 further enhance the usability and adoption of different cluster
 manager.
 
-### Notes/Constraints/Caveats (Optional)
+### Notes/Constraints/Caveats
+#### What's the relationship between the ClusterProfile API and Cluster Inventory?
+The ClusterProfile API represents a single member cluster in a cluster inventory.
+
+#### What's the relationship between a cluster inventory and clusterSet?
+A cluster inventory may or may not represent a ClusterSet. A cluster inventory is considered a [clusterSet](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/1645-multi-cluster-services-api#terminology)
+if all its member clusters adhere to the [namespace sameness](https://github.com/kubernetes/community/blob/master/sig-multicluster/namespace-sameness-position-statement.md) principle.
+In other terms, a clusterSet is a special type of cluster inventory while the reverse is not true.
```

Comment:
I don't think this line ins necessary and I find it a bit confusing. A special type of cluster inventory may represent a cluster set, but cluster set as a concept exists separately from cluster inventory, and is not a special case of CI